data.sql

```diff
@@ -53,3 +53,134 @@ SET owner_id =
         WHEN name = 'Charmander' OR name = 'Squirtle' OR name = 'Blossom' THEN (SELECT id FROM owners WHERE full_name = 'Melody Pond')
         WHEN name = 'Angemon' OR name = 'Boarmon' THEN (SELECT id FROM owners WHERE full_name = 'Dean Winchester')
     END;
+
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/236636710-4ca522a5-5cc4-44a4-a33e-7d1906498360.png)

- [ ] It would be nice if you fix the MIT link to make your readme descriptive and professional 😊